example/App.js

```diff
@@ -96,12 +97,12 @@ export default class App extends Component<Props> {
 
   render() {
     return (
-      <View style={styles.container}>
+      <SafeAreaView style={styles.container}>
```

Comment:
SafeAreaView was new to me! I'm still new at the react-native level of things. This is a nice touch, thanks

Reply:
glad you like it :)